packages/dashboard/src/components/Card/Card.types.ts

```diff
@@ -142,9 +142,9 @@ export interface ICardProps {
 
 export interface ICard extends ICardProps {
   /**
-   * The card key, which must be unique within the dashboard
+   * The card id, which must be unique within the dashboard
    */
-  key: string;
+  id: string;
```

Comment:
> id: string; [](start = 1, length = 12)

Do we really need to do this? This will break the product code

Reply:
I added ICard in my last PR and I thought you are not consuming it yet. Key is a React prop and it was my mistake in the previous PR. We should not add a prop with the name 'key'.  Let me know if there is a bigger impact in the production. 

---
In reply to: [214104426](https://github.com/OfficeDev/office-ui-fabric-react/pull/6153#discussion_r214104426) [](ancestors = 214104426)